tests/lib/store.sh

```diff
@@ -82,8 +82,8 @@ EOF
 }
 EOF
 
-    fakestore new-snap-declaration --dir "$dir" --snap-decl-json=/tmp/snap-decl.json "$snap_path"
```

Comment:
why this change ?

Reply:
It's part of https://github.com/snapcore/snapd/pull/10470, which was recently merged. I'll merge the master branch into this branch, and we should see this change go away.